src/components/Clusters/components/AuthForm.jsx

```diff
@@ -74,6 +76,7 @@ const TokenForm = ({ resource, setResource, ...props }) => {
   const [token, setToken] = useState(resource?.users?.[userIndex]?.user?.token);
 
   useEffect(() => {
+    //eslint-disable-next-line react-hooks/set-state-in-effect
     setToken(resource?.users?.[userIndex]?.user?.token);
   }, [resource, userIndex]);
```

Comment:
I don't see added value on this, please double-check, but in my opinion this just could be removed, but I didn't check kubeconfigID flow.